google_cloudsql_mysql/main.tf

```diff
@@ -0,0 +1,126 @@
+/**
+ * # cloudsql-mysql
+ * Creates CloudSQL MySQL Instance.
+ */
+
+locals {
+  default_database_name = "${var.application}-${var.realm}-${var.environment}-${var.instance_version}"
+  database_name         = coalesce(var.custom_database_name, local.default_database_name)
+  tier                  = coalesce(var.tier_override, "db-custom-${var.db_cpu}-${var.db_mem_gb * 1024}")
+
+  default_replica_name = "${local.database_name}-replica"
+  replica_name         = coalesce(var.custom_replica_name, local.default_replica_name)
+
+  ip_addresses = google_sql_database_instance.primary.ip_address
+
+  enable_ha = var.realm == "prod" ? true : false
+}
+
+resource "google_sql_database_instance" "primary" {
```

Comment:
do we want to add any kind of ability to override `deletion_protection` for when we do want terraform to delete a db (i'd imagine it'd have to be a 2 PR process, one changing the deletion_protection default from true to false, a second to then ask atlantis to actually delete the instance) - or should we just leave deletion_protection outside of our terraform & defaulting to true, so databases have to be manually deleted?

Reply:
added

Reply:
thank you!